src/main/java/ch/njol/skript/expressions/ExprChatRecipients.java

```diff
@@ -113,8 +110,28 @@ public void change(Event event, @Nullable Object[] delta, ChangeMode mode) {
 			case REMOVE_ALL:
```

Comment:
I know that this is the current behavior, but why does `REMOVE_ALL` as `RESET` and `DELETE`? `remove all {_x} from the chat recipients` shouldn't just remove all the recipients. I don't believe it should be supported here at all anyway, as there can only be *one* of a recipient, not more.

Also shouldn't resetting the recipients add all players back in, rather than clearing them? Because that's what I interpret from the `RESET` change mode - to reset the value to how it originally is; in this case all the players

Reply:
Not resolved